### DISCUSSION TITLE
MongoDB with Panache - Pojo Codec Customization

### DISCUSSION THREAD

**Discussion by theseeker58** [quarkusio/quarkus#24575] [Q&A]
Hi there,
using Panache for simplifying MongoDB is it possible to customize Pojo Codec, and how? I'd like to enable USE_GETTERS_FOR_SETTERS for collections, as I did in the past without Panache.
Many thanks

[a comment by an automated account is not shown]

**Comment by loicmathieu** [comment]
@theseeker58 the Pojo Codec is used by MongoDB with Panache and is not customable at the moment.
It is configured directly inside the MongoDB extension: https://github.com/quarkusio/quarkus/blob/main/extensions/mongodb-client/runtime/src/main/java/io/quarkus/mongodb/runtime/MongoClients.java#L332 and it uses the default conversions.

If you need to be able to configure it, please open an enhancement request (as a github issue) describing your use case.
